lib/include/ert/enkf/local_dataset.hpp

```diff
@@ -20,6 +20,10 @@
 #ifndef ERT_LOCAL_DATASET_H
 #define ERT_LOCAL_DATASET_H
 #include <ert/tooling.hpp>
+#include <ert/util/stringlist.hpp>
+#include <ert/util/hash.hpp>
+
```

Comment:
remove empty line

Reply:
There is a conscious grouping; where headers which in some sense belong together are grouped together. In this case the grouping is:
```
#include <standard_headers>

#include <libecl_headers>

#include <libres_headers>
```
but sure - I'll remove all blank lines if that is desired.